pkgs/applications/editors/textadept/deps.nix

```diff
@@ -0,0 +1,42 @@
+{
+    "542782a4df7d.zip" = {
```

Comment:
Was this file automatically generated?

Reply:
no, but i put it in a format where it could easily be

Reply:
i would have committed the script if there was one ofc